sst/test/include/sstsimengtest.hh

```diff
@@ -0,0 +1,5 @@
+#include "framework/macros/eval.hh"
+#include "framework/macros/group.hh"
+#include "framework/registry.hh"
+#include "framework/runner.hh"
+#include "framework/stats.hh"
```

Comment:
Adding to the previous point on this, these headers (IMO) can be added into main.cc also, with this file being removed all together

Reply:
If remove this file, then everything we have define a testcase we would have to import both a `macro/group.h` and `macro/eval.h`. I think unifies the 'include statements needed. However, happy to remove it.

Reply:
I don't think it is too much overhead to add an additional header to each new test file. But again, its a very minor thing so you can leave it as is if you prefer

Reply:
added extra file framework/parser.hh which is needed by each test, I vote to keep the single header as the inclusion of parser now means, every test file now needs to include 3 files.